mono/mini/interp/interp-internals.h

```diff
@@ -137,7 +137,11 @@ struct _InterpFrame {
 	 * Storing into this field will keep the object pinned
 	 * until the objref can be stored into stackval->data.o.
 	 */
+#ifdef TARGET_WASM
 	MonoObject* volatile o;
+#else
```

Comment:
This should go also for `stackval` structure. Well I guess only there, since `o` is dead here.